backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5591490_sys_gh11244_Attribute_IsAlwaysUpdatable.sql

```diff
@@ -0,0 +1,522 @@
+;
+
+-- 2021-06-09T08:59:20.709Z
+-- URL zum Konzept
+INSERT INTO AD_Field_Trl (AD_Language,AD_Field_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language, t.AD_Field_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Field t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y') AND t.AD_Field_ID=647478 AND NOT EXISTS (SELECT 1 FROM AD_Field_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Field_ID=t.AD_Field_ID)
+;
+
+-- 2021-06-09T08:59:20.745Z
+-- URL zum Konzept
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(2468) 
+;
+
+-- 2021-06-09T08:59:20.801Z
+-- URL zum Konzept
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=647478
+;
+
+-- 2021-06-09T08:59:20.837Z
+-- URL zum Konzept
+/* DDL */ select AD_Element_Link_Create_Missing_Field(647478)
+;
+
+-- 2021-06-09T09:01:51.269Z
+-- URL zum Konzept
+INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_ElementGroup_ID,AD_UI_Element_ID,AD_UI_ElementType,Created,CreatedBy,Description,Help,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayedGrid,IsDisplayed_SideList,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNoGrid,SeqNo_SideList,Updated,UpdatedBy) VALUES (0,647478,0,462,540209,585690,'F',TO_TIMESTAMP('2021-06-09 12:01:50','YYYY-MM-DD HH24:MI:SS'),100,'The column is always updateable, even if the record is not active or processed','If selected and if the winow / tab is not read only, you can always update the column.  This might be useful for comments, etc.','Y','N','N','Y','N','N','N',0,'Always Updateable',70,0,0,TO_TIMESTAMP('2021-06-09 12:01:50','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2021-06-09T09:03:40.041Z
+-- URL zum Konzept
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,Created,CreatedBy,Description,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,579314,0,TO_TIMESTAMP('2021-06-09 12:03:39','YYYY-MM-DD HH24:MI:SS'),100,'When this flag is set on true it means the attribute''s value can be changed even after the handling unit is no longer in the "Palling" status.','D','Y','Attribute Always Updatable','Attribute Always Updatable',TO_TIMESTAMP('2021-06-09 12:03:39','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
that's not true, as you documented. please find a description that tells the used what to expect and not expect when working with this flag

Reply:
done